closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -70,6 +77,7 @@ final class NativeGaloisCounterMode extends FeedbackCipher {
     // java byte array, e.g. Integer.MAX_VALUE, since all data
     // can only be returned by the doFinal(...) call.
     private static final int MAX_BUF_SIZE = Integer.MAX_VALUE;
+    private static final byte[] EMPTY_BUF = new byte[0];
```

Comment:
Suggest you move this earlier so it can be a (more general) replacement for `emptyAAD` as well.

Reply:
Changed.

Reply:
Please remove `emptyAAD`, replacing references with `EMPTY_BUF`.

Reply:
Oh, you mean replace every occurrence of `emptyAAD` with `EMPTY_BUF`.

Reply:
Done.